src/pages/components/pagination.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Pagination
+description: >
+  Pagination separates content into pages and allows users to navigate between those pages.
+package: '@zendeskgarden/react-pagination'
+components:
+  - pagination/src/elements/Pagination/Pagination.tsx
+  - pagination/src/elements/CursorPagination/CursorPagination.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To divide large amounts of data into manageable chunks</li>
+    </ul>
+  </Use>
+</Usage>
+
+## How to use it
+
+### Cursor pagination
+
+Cursor-based pagination works by returning a pointer to a specific item in a
+dataset. There is no concept of the total number of pages or results in the set.
+
+import CursorDefault from '../../examples/pagination/CursorDefault.tsx';
+import CursorDefaultCode from '!!raw-loader!../../examples/pagination/CursorDefault.tsx';
+
+<CodeExample code={CursorDefaultCode}>
+  <CursorDefault />
+</CodeExample>
+
+### Gap indicator
+
+The position of a gap indicator (...) can be set in the sequence to represent overflowing pages.
+
+import OffsetGap from '../../examples/pagination/OffsetGap.tsx';
+import OffsetGapCode from '!!raw-loader!../../examples/pagination/OffsetGap.tsx';
+
+<CodeExample code={OffsetGapCode}>
+  <OffsetGap />
+</CodeExample>
+
+### Page padding
+
+The amount of page numbers to display (not including page gap indicators or
+arrows) is modified with `pagePadding`. Each increment adds 2 page numbers.
```

Comment:
This description is not accurate. Page padding sets the number of pages that may appear between the current page and a gap. For example, one could set `pagePadding={0}`.

Reply:
Nice, that makes way more sense. Updated.